platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/K2IntegrationTest.kt

```diff
@@ -137,8 +136,6 @@ class K2IntegrationTest : AbstractKotlinIntegrationTest() {
 
     private
     fun assertCanConsumeK2BuildLogic() {
-        gradleExecuterFor(arrayOf("help"))
-            .expectDeprecationWarning("w: The `kotlin.experimental.tryK2` deprecated property is used in your build.")
-            .run()
+        build("help")
```

Comment:
❌ We must keep the `gradleExecuterFor(arrayOf("help")).expectDeprecationWarning("w: The `kotlin.experimental.tryK2` deprecated property is used in your build.").run()` here